semantics.json

```diff
@@ -174,6 +174,24 @@
                         "label": "License"
                       }
                     ]
+                  },
+                  {
+                    "name": "nextPreviousSlide",
+                    "type": "select",
+                    "label": "Go To Next or Previous Slide",
+                    "description": "This will override the Go To Slide Number",
+                    "importance": "low",
+                    "optional": true,
+                    "options": [
+                      {
+                        "value": "next",
+                        "label": "Next"
+                      },
+                      {
+                        "value": "previous",
+                        "label": "Previous"
+                      }
```

Comment:
Maybe make the dropdown non-optional and:

1. Name it "Go to" with three options, "Specified slide", "Previous slide" and "Next slide"
2. Make "Specified slide" the default

?